src/coremods/core_oper/core_oper.h

```diff
@@ -41,11 +33,18 @@ namespace DieRestart
  */
 class CommandDie : public Command
 {
+ protected:
+	bool CheckPass(User* user, const std::string& inputpass) const;
+
  public:
+	std::string hash, password;
+
 	/** Constructor for die.
 	 */
 	CommandDie(Module* parent);
 
+	CommandDie(Module* parent, const std::string& cmd);
```

Comment:
What is this second constructor for? It doesn't appear to be used anywhere.

Reply:
it is used by CommandRestart, since that is a subclass of CommandDie